workspace_tools/toolchains/gcc.py

```diff
@@ -44,7 +46,7 @@ def __init__(self, target, options=None, notify=None, macros=None, silent=False,
         if target.core.startswith("Cortex"):
             self.cpu.append("-mthumb")
 
-        if target.core == "Cortex-M4F":
+        if target.core == "Cortex-M4F" or target.core == "Cortex-M7F":
```

Comment:
Also, I remember there are two FPU implementations Simple and Double, aren't they?